File: amazia-core/src/main/java/com/example/delivery/controller/GetDeliveryController.java
package com.example.delivery.controller;

import com.example.delivery.dto.DeliveryResponse;
import com.example.delivery.service.GetDeliveryService;
import org.springframework.http.ResponseEntity;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.PathVariable;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

/**
 * 配送詳細 API（GET /api/deliveries/{id}）。
 */
@RestController
@RequestMapping("/api")
public class GetDeliveryController {

    private final GetDeliveryService service;

    public GetDeliveryController(GetDeliveryService service) {
        this.service = service;
    }

    @GetMapping("/deliveries/{id}")
    public ResponseEntity<DeliveryResponse> get(@PathVariable Long id) {
        return ResponseEntity.ok(service.get(id));
    }
}

File: amazia-console/app/Delivery/Controller/GetDeliveryController.php
<?php

namespace App\Delivery\Controller;

use App\Delivery\Service\GetDeliveryService;
use App\Http\Controllers\Controller;

/**
 * 配送詳細 Controller（GET /api/deliveries/{id}）。
 */
class GetDeliveryController extends Controller
{
    private GetDeliveryService $service;

    public function __construct(GetDeliveryService $service)
    {
        $this->service = $service;
    }

    public function __invoke(int $id)
    {
        $response = $this->service->get($id);
        return response()->json($response->json(), $response->status());
    }
}
